app/views/Licenses.js

```diff
@@ -35,7 +38,9 @@ export const LicensesScreen = ({ navigation }) => {
   };
 
   const handleTermsOfUsePressed = () => {
-    Linking.openURL(PRIVACY_POLICY_URL);
+    Linking.openURL(
+      i18n.language === 'es' ? PRIVACY_POLICY_URL_ES : PRIVACY_POLICY_URL,
```

Comment:
You are using this line of code twice in this file, I think is better create a variable with the two option and use it where its need